src/notifications/generate.ts

```diff
@@ -72,6 +76,8 @@ export const notificationTitleMap: Record<
     `<b>${ctx.doneBy.name}</b> <span class="text-theme-color-cabbage">viewed</span> your post on <b>${ctx.source.name}</b>.`,
   squad_blocked: (ctx: NotificationSourceContext) =>
     `You are no longer part of <b>${ctx.source.name}</b>`,
+  squad_subscribe_to_notification: (ctx: NotificationSourceContext) =>
+    `Congrats on your first post on <b>${ctx.source.name}</b>. Subscribe to get updates about activity in your squad.  Subscribe for updates.`,
```

Comment:
Asked a [question](https://dailydotdev.slack.com/archives/C0506KU4RH9/p1683088513646799) to sab now, this copy seems a bit off.

Also did you notice the small copy is different, not sure we support it, but worth checking maybe?

<img width="418" alt="Screenshot 2023-05-03 at 06 35 25" src="https://user-images.githubusercontent.com/554874/235833600-6efa9489-249c-4858-acf8-18cb80ab1eb6.png">

Reply:
Oh, they might have changed it recently (or the design have a different text on in-app and notifications page). It used to have the Squad name in the text, hence, I prefixed the type with `squad` and used the source object. Will check this with them.

Reply:
Oh, nice. Thanks for asking it to Sab. I think they should be the same as it is the same entity when given back.

Reply:
Maybe just add that question in the thread whether those can be the same copy :)